### ISSUE TITLE
Add SwiftUI Support

### ISSUE THREAD

**Issue by colintheshots** [kuuuurt/multiplatform-paging#24]
The current implementation for iOS only works with UiKit for populating a paged list. However, many projects have switched entirely over to SwiftUI for their UI implementation.

It would be nice to have a working SwiftUI example for this library.

**Comment by kuuuurt** [comment]
Hi @colintheshots. @joreilly has a great example for this! Here's the [link](https://github.com/joreilly/MortyComposeKMM/blob/main/iosApp/iosApp/Features/Characters/CharacterListViewModel.swift)

**Comment by colintheshots** [comment]
Nice. That example or a very similar one should be integrated into this repository so others can find it.

**Comment by kuuuurt** [comment]
@colintheshots i agree! for now, i added a section for Compose and SwiftUI linking to the repo for now. I'll add an own example in the future when the time allows me to but for now, I think this can point people in the right direction. 🙂
